### ISSUE TITLE
When opponent's Barnes pulls a card from their deck, that card should appear in their deck list, greyed out

### ISSUE THREAD

**Issue by garyking** [HearthSim/Hearthstone-Deck-Tracker#3062]
When the opponent plays a Barnes and it pulls a card from their deck, the opponent's deck list should therefore show the pulled card as one that is in their deck, therefore showing it but greyed out.

**Comment by judge2020** [comment]
Barnes does not pull a minion out of the deck, it summons a **copy** of it and the opponent can play that card later when they draw it. 

(if this isn't already a thing) it should put the card in the opponent's deck as it does with joust.

**Comment by garyking** [comment]
Yes that's what I meant. I know that it makes a 1/1 copy. So that's a clue that that minion is in the deck, and HDT should reflect that.
